math/NTPoly/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           mpi 1.0
+
+github.setup        william-dawson NTPoly 2.7.1 ntpoly-v
+categories          math
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Library for computing the functions of sparse matrices
+long_description    NTPoly is a massively parallel library for computing the functions of sparse, \
+                    Hermitian matrices based on polynomial expansions. For sufficiently sparse matrices, \
+                    most of the matrix functions in NTPoly can be computed in linear time.
+homepage            https://william-dawson.github.io/NTPoly
+checksums           rmd160  f9c072768f4644fa0aafbf7dd79f14045d17de6e \
+                    sha256  6b941d0e37bb7e047ad7d866e5dce842f63d3da6c60d3c2299e78cb1c05918e1 \
+                    size    281974
+
+depends_lib-append  path:lib/libopenblas.dylib:OpenBLAS
+
+if {${os.platform} eq "darwin" && ${os.arch} eq "powerpc"} {
+    mpi.setup       require require_fortran \
+                    -gcc44 -gcc45 -gcc46 -gcc47 -gcc48 -gcc49 -gcc5 -gcc6 \
+                    -clang -fortran
```

Comment:
I don’t quite understand how it is that having both “require_fortran” and “-fortran” makes sense here… can you explain why that combo is what is needed?

Reply:
@kencu TBH that was borrowed from another mpich-using port as-is. Let me try without `-fortran`.

Reply:
Yeah - PortGroups... did I ever tell you how much I dislike PortGroups?

Probably 1 person in MacPorts might know what is going on here... and it is not me.

Reply:
Completely agree here. It is a mess.

For now, until we sort `mpi` PG out for PPC, I suggest leaving the code as-is, since it works fine for practical purposes, both on x86 and PPC. Once I have `mpi-default` fixed, I gonna remove this `-fortran` line from `mpi.setup` in the few ports where it is used presently.
(There are other math ports which need this, but I do not want to commit something that gonna be broken for PPC until a later, indefinite date.)